### DISCUSSION TITLE
error on tfs when custom namespace is applied

### DISCUSSION THREAD

**Discussion by gbr1** [iRobotEducation/create3_docs#126] [Questions] [answered]
I don't know if this is the right place to submit an issue, but when you set a namespace on create, `base_footprint` and `odom` frames are not more present in the tf_tree

Thank you.

**Comment by justinIRBT** [comment, marked as answer]
See the discussion here:
https://github.com/iRobotEducation/create3_docs/discussions/39

**Reply by gbr1** [reply]
thx sorry for duplicating
